text/0000-grid-inventory.md

```diff
@@ -0,0 +1,126 @@
+- Feature Name: (fill me in with a unique identifier, my_awesome_feature)
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+- Hyperledger Grid Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC proposes a generic and extensible framework for a Hyperledger Grid
+Inventory component, and is inspired by GS1’s EDI Inventory Report - Business
+Message Standard. GS1 is a widely used data standard in enterprises and, given
+that positioning and familiarity, Grid support feels natural. Additional
+implementations of Grid Inventory may derive from or extend this
+implementation.
+
+Grid Inventory leverages other extensible Hyperledger Grid components
+including: Grid Product, and Grid Location. Grid Inventory aims to provide
+current inventory quantity & status visibility for each Grid Product and Grid
+Location defined within the Grid Inventory smart contract between parties, as
```

Comment:
The _Summary_ should prob just get a re-read/edit after it's sat for a while, but this reference to "smart contract between parties" is confusing to me.  I'm guessing we should just remove it?

Reply:
Thank you Dave.   We can remove.